sdk/cosmos/src/requests/create_slug_attachment_builder.rs

```diff
@@ -56,7 +57,7 @@ impl<'a, 'b, ContentTypeSet> CreateSlugAttachmentBuilder<'a, 'b, No, ContentType
 where
     ContentTypeSet: ToAssign,
 {
-    pub fn body(self, body: &'b [u8]) -> CreateSlugAttachmentBuilder<'a, 'b, Yes, ContentTypeSet> {
+    pub fn body(self, body: Bytes) -> CreateSlugAttachmentBuilder<'a, 'b, Yes, ContentTypeSet> {
```

Comment:
I'm worried that having this be `Bytes` makes it harder for the user to use. Making this `Into<Bytes>` might help though